p2panda-rs/src/test_utils/utils.rs

```diff
@@ -0,0 +1,124 @@
+
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+#![allow(missing_docs)]
+use serde::Serialize;
+
+use crate::hash::Hash;
+use crate::entry::{Entry, LogId, SeqNum};
+use crate::identity::KeyPair;
+use crate::message::{Message, MessageFields, MessageValue};
+
+// A custom `Result` type to be able to dynamically propagate `Error` types.
+pub type Result<T> = std::result::Result<T, Box<dyn std::error::Error>>;
+
+
+pub const MESSAGE_SCHEMA: &str  = "00401d76566758a5b6bfc561f1c936d8fc86b5b42ea22ab1dabf40d249d27dd906401fde147e53f44c103dd02a254916be113e51de1077a946a3a0c1272b9b348437";
```

Comment:
How is this different from `MESSAGE_SCHEMA` in `fixtures/utils.rs`?

Reply:
Yeh, they're the same! I kept them in `test_utils/utils.rs` so they can be imported into test and non-test modules.